apps/admin/backend/src/store.ts

```diff
@@ -235,6 +237,39 @@ export class Store {
     return settings?.currentElectionId ?? undefined;
   }
 
+  /**
+   * Creates a system settings record and returns its ID.
+   * Note `system_settings` are logical settings that span other machines eg. VxScan.
+   * `settings` are local to VxAdmin
+   */
+  addSystemSettings(systemSettings: SystemSettings): void {
```

Comment:
What do you think of renaming this `setSystemSettings` or `saveSystemSettings`, since `addSystemSettings` loosely implies that there can be multiple system settings objects?

Reply:
Agree, changed in https://github.com/votingworks/vxsuite/pull/3147/commits/d6b5efd7f572c85c2a475a53edd0db0c76de2cc6